packages/syft/src/syft/core/node/enums.py

```diff
@@ -37,3 +40,8 @@ class ResponseObjectEnum(str, Enum):
     USER = "user"
     WORKER = "worker"
     DATA = "data"
+
+
+class TensorPointerStatus(str, Enum):
```

Comment:
MAYBE: we could keep as PointerStatus, as we use them with several pointers , like phi,gamma also?

Reply:
Solved by: [8164bae](https://github.com/OpenMined/PySyft/pull/7105/commits/8164bae204debcfdeaf15d377c61bf68f7f5ef2b)